.eslintrc.yml

```diff
@@ -27,4 +27,17 @@ rules:
     - error
     - skipUndeclared: true
 settings:
-  import/resolver: webpack
+  import/resolver:
```

Comment:
Duplicating the config here seems to be a necessary evil due to https://github.com/tleunen/eslint-import-resolver-babel-module/issues/95 (in order for eslint to properly resolve the modules). This is fixed on `5.0.0` of `eslint-import-resolver-babel-module`, but we're locked to `^4.0.0` because 5 requires Babel 7, and upgrading Babel feels outside the scope of this work.